SUMMARY.md

```diff
@@ -16,6 +16,8 @@
   * [Images](distributor-api-v1/images.md)
   * [Payment Gateway Data](distributor-api-v1/payment-gateway-data.md)
   * [Environments](distributor-api-v1/environments.md)
+  * [Deprecations](distributor-api-v1/deprecations/README.md)
```

Comment:
Not sure if this is a good place, It's really hidden. I am afraid that people will skip it. I would reference it maybe on the top of the change log page.